draft-ietf-quic-multipath.md

```diff
@@ -101,8 +101,10 @@ migration.
   * Use the same packet header formats as QUIC version 1 to avoid the
 risk of packets being dropped by middleboxes (which may only support
 QUIC version 1)
-  * Congestion Control, RTT measurements and PMTU discovery should be
-per-path (following {{QUIC-TRANSPORT}})
+  * Congestion Control must be per-path (following {{QUIC-TRANSPORT}})
+which usually also requires per-path RTT measurements
+  *  PMTU discovery should be performed per-path, however, the minimum
+PMTU of all paths may be used on each paths
```

Comment:
I would remove "however, the minimum PMTU of all paths may be used on each paths". Implementations can always send packets shorter than the path MTU, so we are not saying much here. Also, there is a minimum path MTU for QUIC, and implementations that don't do PMTUD can always use that, so again, no need to be too specific.

Reply:
If I remember correctly, the purpose of the sentence, "minimum PMTU of all paths may used ..." is to say when forming a packet before inserting it in the send_queue, it's better to constrain the size of the packet so that it can be scheduled to any of the paths for the sake of flexibility in scheduling. For example, when we want to use re-injection in our packet scheduling, we need to make sure that a packet sent on path 1 can also be reinjected on path 2. Therefore, what we did was to use per-path PMTU discovery and the use the minimum PMTU of all paths for each individual path. But indeed, this choice may vary in a different implementation.

Reply:
That's completely implementation specific. Picoquic, for example, schedules the path first and then formats the packet. There is no need to go to that level of detail in the specification.

Reply:
@yfmascgy yes the intention of adding this what to give a hit towards the implementation option you mentioned. However, as you both say this is implementation specific and doesn't need to be in the intro part. So I'm fine removing this and already create a change request for this where I only need to push "commit".

If we want to discussion these implementation options more, we could add text in the "implementation consideration" section. @yfmascgy feel free to prose a PR for that or we could open a new issue to address this later.

Reply:
Yes. I agree removing this from the intro and we can discuss this in the implementation consideration.

Reply:
@mirjak I have submitted PR #105 for the PMTU implementation considerations.